include/red_black_tree.h

```diff
@@ -68,14 +73,22 @@ class RedBlackTree : public BinarySearchTree<K, V> {
         return true;
     }
 
+    bool remove(const K& key) {
```

Comment:
It's better to add `override` for the overriding method:
```C++
bool remove(const K& key) override;
```

Reply:
remove is template method.

Reply:
OK, I SEE.

Reply:
Only virtual function can be override.